src/main/java/world/inetum/realdolmen/inetumrealJobs/entities/User.java

```diff
@@ -52,22 +48,20 @@
     @NotBlank
     @Column(name = "city")
     private String city;
+    @NotBlank
     @Column(name = "postal_code")
     private String postalCode;
     @Column(name = "country")
     @NotBlank
     private String country;
     @Column(name = "mobile_phone")
     private String mobilePhone;
-    @NotBlank
     @Column(name = "profile_picture")
     private String profilePicture;
 
-    @OneToMany(fetch = FetchType.LAZY)
-    @JoinTable(	name = "user_roles",
-            joinColumns = @JoinColumn(name = "user_id"),
-            inverseJoinColumns = @JoinColumn(name = "role_id"))
-    private Set<Role> roles = new HashSet<>();
+    @Column(name ="role")
+    @Enumerated(EnumType.STRING)
+    private ERole role;
```

Comment:
Now that Role is gone, rename ERole to Role

Reply:
Fixed